handle_msg_middleware.go

```diff
@@ -26,3 +26,14 @@ func (h *handler) getHandleCallMsgNestedware(cp *callProc) HandleCallMsgFunc {
 	}
 	return nestedWare
 }
+
+// PreventMessagesWithouID is a HandleCallMsgMiddleware for preventing messages without ID
+var PreventMessagesWithouID HandleCallMsgMiddleware = func(next HandleCallMsgFunc) HandleCallMsgFunc {
```

Comment:
How about naming this middleware as "ValidateJsonRpc" by checking `message.method` and `message.Id` directly?